src/SUMMARY.md

```diff
@@ -144,6 +144,7 @@
     - [ADTs and Generic Arguments](./ty_module/generic_arguments.md)
     - [Parameter types/consts/regions](./ty_module/param_ty_const_regions.md)
 - [`TypeFolder` and `TypeFoldable`](./ty-fold.md)
+- [Normalization and Aliases](./normalization/normalization.md)
```

Comment:
maybe we should not nest it this deep, unless the plan is to add other files under normalization/ dir